### PR TITLE
Revamp auto sequences

### PR DESCRIPTION
**Pull request by rutmanz**
- Highlight selected auto paths on Shuffleboard
- Make path management cleaner with `AutoPath`
- Add support for custom setpoints
- Spin up flywheels while driving
- Don't waste time trying to shoot a second ball if only one is staged
- Add Auto1Ball that doesn't taxi

### CODE REVIEW

File: src/main/java/org/team1540/robot2022/utils/AutoHelper.java [hunk 1]
```diff
@@ -113,4 +151,27 @@ public static Trajectory getTrajectory(String trajectoryName) {
...
+        public double frontSetpoint, rearSetpoint;
+        public String name;
+        
+        public static AutoPath auto1Ball   = new AutoPath("2ball.posA.path1.wpilib.json", 2200, 2500, true);
+        public static AutoPath auto2Ball1A = new AutoPath("2ball.posA.path1.wpilib.json", 2200, 2500, true);
+        public static AutoPath auto2Ball1B = new AutoPath("2ball.posB.path1.wpilib.json", 2200, 2500, true);
+        public static AutoPath auto3Ball2 = new AutoPath("3ball.posA.path2.wpilib.json", 2500, 2800, true);
+        public static AutoPath auto4Ball2 = new AutoPath("4ball.posA.path2.wpilib.json", 2400, 2800, true);
+        public static AutoPath auto5Ball2 = new AutoPath("5ball.posA.path2.wpilib.json", 2600, 2900, true);
+        public static AutoPath auto5Ball3 = new AutoPath("5ball.posA.path3.wpilib.json", 2400, 2800, true);
```
Comment (natesales) [inline src/main/java/org/team1540/robot2022/utils/AutoHelper.java:167]: Where did these flywheel speeds come from?

**Comment by natesales** [comment]
This is really great!

File: src/main/java/org/team1540/robot2022/utils/AutoHelper.java (hunk 1 above)
Comment (rutmanz) [reply src/main/java/org/team1540/robot2022/utils/AutoHelper.java:167]: They came from me guessing at good starting points, they definitely need to be tuned

Comment (natesales) [reply src/main/java/org/team1540/robot2022/utils/AutoHelper.java:167]: Got it. #103

**Review by natesales — APPROVED** [review]

**Pull request merged**